docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04 as web-base
+
+RUN apt update && apt install -y git curl
+RUN curl -sL https://deb.nodesource.com/setup_14.x -o nodesource_setup.sh && \
+    bash ./nodesource_setup.sh && \
+    apt install nodejs && \
+    rm ./nodesource_setup.sh
+
+FROM web-base as web-sdk-builder
+RUN apt update
+# checkout repo
+RUN mkdir -p /usr/local/src && \
```

Comment:
Since Docker layer caching might use a cached layer even if the v-web sources have changed, adding a `ADD https://api.github.com/repos/vircadia/vircadia-web/git/refs/heads/master vircadia-web-git-version.json` before this RUN will break the cache if the sources have changed.

Reply:
Adding this line causes an error like following:
==============================
21 40.93 npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
#21 41.21 npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
#21 50.11 npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
#21 71.51 npm ERR! code ERR_SOCKET_TIMEOUT
#21 71.51 npm ERR! network Socket timeout
#21 71.51 npm ERR! network This is a problem related to network connectivity.
==========================
Any idea why?

Reply:
I saw those errors also but figured they were related to v-web-sdk configuration/versioning. Maybe need a revisit of the v-web-sdk build independent of the Docker image. Wonder what v-web-sdk has for package-lock and other requirements.